pkg/base-dev/lib/story.hoon

```diff
@@ -0,0 +1,109 @@
+/-  *story
```

Comment:
These base-dev files need to be symlinked into pkg/arvo

Reply:
done in https://github.com/urbit/urbit/pull/5612/commits/add4452589f40146fe2e93ded4785b3f86eb50cc